storage/innobase/srv/srv0start.cc

```diff
@@ -347,6 +355,12 @@ inline dberr_t trx_sys_t::reset_page(mtr_t *mtr)
   mtr->write<2>(*sys_header, FIL_PAGE_TYPE + sys_header->page.frame,
                 FIL_PAGE_TYPE_TRX_SYS);
 
+  mtr->write<4>(*sys_header,
+		TRX_SYS + TRX_SYS_FSEG_HEADER + 4 +
+                sys_header->page.frame, trx_fseg_addr.page);
+  mtr->write<2>(*sys_header,
+		TRX_SYS + TRX_SYS_FSEG_HEADER + 8 +
+                sys_header->page.frame, trx_fseg_addr.boffset);
   mtr->write<4>(*sys_header,
```

Comment:
Please do not use TAB for indentation.

We should use one of the `mtr_t::memcpy()` functions to write these 6 consequent bytes, instead of writing two `WRITE` records.